content/concept-transaction-cost.md

```diff
@@ -69,9 +69,24 @@ The `rippled` server uses a variety of heuristics to estimate which transactions
+* A single sending address can have at most 10 transactions queued at the same time. In order for a transaction to be queued, the sender must have enough XRP to pay all the XRP costs of all the sender's queued transactions including both the `Fee` fields and the sum of the XRP that each transaction could send. [New in: rippled 0.32.0][]
+* If a transaction affects how the sending address authorizes accounts, no other transactions from the same address can be queued behind it. [New in: rippled 0.32.0][]
 * If the transaction includes a `LastLedgerSequence` field, the value of that field must be at least **the current ledger index + 2**.
 
+#### Fee Averaging ####
+
+[New in: rippled 0.33.0][]
+
+If a sending address has one or more transactions queued, that sender can "push" the existing queued transactions into the open ledger by submitting a new transaction with a high enough transaction cost to pay for all of them. Specifically, the new transaction must increase the average transaction cost of the queued transactions from the same sending address, including itself, so that each of those transactions would be included in the open ledger if the `Fee` value of each of those transactions were equal to the average. The transactions must still follow the other [queuing restrictions](#queuing-restrictions) and the sending address must have enough XRP to pay the transaction costs of all the queued transactions.
+
+This feature helps you work around a particular situation. If you submitted one or more transactions with a low cost, which got queued, you cannot send new transactions from the same address unless you do one of the following:
+
+* Wait for the queued transactions to be included in a validated ledger, _or_
+* [Cancel the queued transactions](reference-transaction-format.html#canceling-or-skipping-a-transaction) by submitting a new transaction with the same sequence number.
+
+If the transaction cost of the queued transactions is low, you may have to wait a long time for a validated ledger to include the queued transactions, because other senders can "cut in line" by submitting transactions with higher transaction costs. Since signed transactions are immutable, you cannot increase the transaction cost of the queued transactions to increase their priority. If you do not want to invalidate the previously submitted transactions, fee averaging provides a workaround. If you increase the transaction cost of your new transaction to compensate, you can ensure the queued transactions are included in a validated ledger right away.
```

Comment:
Define "long time". In practice, at least so far, queued transactions don't usually have to wait more than one or two ledgers. Or are you just leaving it intentionally vague to emphasize the risk?

> you can ensure the queued transactions are included in a validated ledger right away.

Pedantically, "you can ensure the queued transactions are included in an open ledger right away.".